packages/@expo/cli/src/start/startAsync.ts

```diff
@@ -72,12 +73,16 @@ export async function startAsync(
   if (exp.platforms?.includes('ios') && process.platform !== 'win32') {
     // If Xcode could potentially be used, then we should eagerly perform the
     // assertions since they can take a while on cold boots.
-    getXcodeVersionAsync({ silent: true });
+    await getXcodeVersionAsync({ silent: true });
     SimulatorAppPrerequisite.instance.assertAsync().catch(() => {
       // noop -- this will be thrown again when the user attempts to open the project.
     });
   }
 
+  if (exp.platforms?.includes('android')) {
+    await AndroidPrerequisite.instance.assertAsync();
+  }
```

Comment:
I don't think we should assert `adb` is installed on `expo start`. I think `expo run android` definitely requires `adb`, but I think you don't actually need `adb` to start the app using Expo Go, right?

Have you tested the performance too? Might be good to understand if this adds 1s to the startup time, or if it's 100ms 😄